src/react-share.jsx

```diff
@@ -2,10 +2,12 @@ import { generateIcon } from './generateIcon';
 import * as ShareButtons from './share-buttons';
 import * as ShareCounts from './share-counts';
 import * as shareLinks from './social-media-share-links';
+import * as shareCountGetters from './share-count-getters';
 
 export {
   generateIcon as generateShareIcon,
   ShareButtons,
   ShareCounts,
   shareLinks,
+  shareCountGetters,
```

Comment:
I exported this because I don't want to show counter on my site but I want use it just for analitics. So I can call shareCountGetters to know if share is completed or not. Because close callback is called everytime when window is closed I don't know if user finished share or just closed popup.